### PR TITLE
update virtio crates

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: src/device/mmio.rs [hunk 1]
```diff
@@ -67,7 +67,7 @@ impl MmioDeviceSpace {
             version: MMIO_VERSION,
             device_id: device.device_type(),
             vendor_id: VENDOR_ID,
-            device_features: device.device_features(device.device_features_select()),
+            device_features: device.device_features() as u32,
```
Comment (Mic92) [inline src/device/mmio.rs:70]: Not sure about this one.

File: src/device/virtio/block/device.rs [hunk 2]
```diff
@@ -136,32 +138,32 @@ where
...
-impl<M: GuestAddressSpace + Clone + Send + 'static> WithDeviceOps for Block<M> {
+impl<M: GuestAddressSpace + Clone + Send + 'static> VirtioDeviceActions for Block<M> {
     type E = Error;
 
     fn activate(&mut self) -> Result<()> {
         if self.virtio_cfg.device_activated {
             return Err(Error::AlreadyActivated);
         }
 
-        if !self.queues_valid() {
```
Comment (Mic92) [inline src/device/virtio/block/device.rs:161]: And this one. Please check!

File: src/device/mmio.rs (hunk 1 above)
Comment (pogobanane) [reply src/device/mmio.rs:70]: is fine

File: src/device/virtio/block/device.rs (hunk 2 above)
Comment (pogobanane) [reply src/device/virtio/block/device.rs:161]: should be fine. But now that all dependent crates are updated, this code probably doesn't work anyways.

**Pull request merged**